package.json

```diff
@@ -4,13 +4,18 @@
   "scripts": {
     "style": "prettier --write .",
     "style-check": "prettier --check .",
-    "common": "cd common && npm run build && cd .. & npm upgrade common",
-    "build": "cd common && npm run build && cd ../backend && npm run build && cd ../frontend && npm run build",
-    "backend": "cd backend && npm run dev",
-    "frontend": "cd frontend && npm run dev"
+    "build:backend": "cd backend && npm run build",
+    "build:common": "cd common && npm run build && cd .. && npm upgrade common",
+    "build:frontend": "cd frontend && npm run build",
```

Comment:
Just FYI, the `"build:<folder-name>"` pattern is completely arbitrary. These scripts could be called anything, as long as I'm consistent. I just figured the pattern of `command:folder` made sense